t/lib/NAR/Helper.pm

```diff
@@ -138,7 +141,7 @@ sub connect {
     }
 
     $self->_ok( sub {
-        $self->mq->connect( $self->{host}, $options );
+        $self->mq->connect( $self->{host}, $options, $client_properties );
```

Comment:
Can you add a new test here instead of just changing this one? We should make sure both cases are covered.

Reply:
Good point. I've now restored the helper connection function and split out a new test that specifically tests the connection with the client properties site.

Thanks